src/device-api/v1.js

```diff
@@ -25,37 +28,41 @@ export const createV1Api = function (router, applications) {
 		if (appId == null) {
 			return res.status(400).send('Missing app id');
 		}
-		return applications
-			.getCurrentApp(appId)
-			.then(function (app) {
-				let service = app?.services?.[0];
-				if (service == null) {
+
+		// FIX-ME: This cannot work, since we will not know which appId is the one to use
```

Comment:
I don't think this comment is true, because we get the app id from the request (also I tested it  :P)